packages/contracts/src/healthcheck/LossRatioHealthCheck.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.19;
+
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+import {SafeInitializable} from "../upgradeable/SafeInitializable.sol";
+import {IVersionable} from "../upgradeable/IVersionable.sol";
+import {IHealthCheck} from "./IHealthCheck.sol";
+import {SafeUUPSUpgradeable} from "../upgradeable/SafeUUPSUpgradeable.sol";
+
+    error HealthCheckFailed();
+
+contract LossRatioHealthCheck is SafeUUPSUpgradeable, IHealthCheck {
+    event ShutdownLossRatioChanged(uint256 ratio);
+    event HealthCheckEnabledChanged(bool enabled);
+
+    // represents 100%
+    uint256 public constant MAX_BPS = 10_000;
+
+    uint8 public constant PASS = 0;
+    uint8 public constant ACCEPTABLE_LOSS = 1;
+    uint8 public constant SIGNIFICANT_LOSS = 2;
```

Comment:
Can you move this constants to `IHealthCheck.sol`? I think it possible to define constants on file level

Reply:
That is the interface so it doesn't support constants. I can convert it to an abstract class but it will be a bit weird. Another option is to have a separate contract with constants only.

Reply:
Nope, I was wrong it works on file level.